genesyscloud/resource_genesyscloud_journey_segment_test.go

```diff
@@ -0,0 +1,104 @@
+package genesyscloud
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/mypurecloud/platform-client-sdk-go/v72/platformclientv2"
+)
+
+func TestAccResourceJourneySegmentSession(t *testing.T) {
+	runTestCase(t, "basic_session_attributes")
+}
+
+func TestAccResourceJourneySegmentCustomer(t *testing.T) {
+	runTestCase(t, "basic_customer_attributes")
+}
+
+func TestAccResourceJourneySegmentContextOnly(t *testing.T) {
```

Comment:
This fails for me with `API Error: 400 - Event type was not found with name: OtherEventName`

Reply:
Getting `API Error: 400 - Error validating JourneyPatterns. `attributes.bleki.value` is not a valid attribute for eventType: All Event Types (01ed790c-db26-452c-9834-effb75599daf)`